pkg/embeddedcluster/upgrade.go

```diff
@@ -75,6 +76,11 @@ func startClusterUpgrade(
 
 	log.Printf("Starting cluster upgrade to version %s...", newcfg.Version)
 
+	// We cannot notify the upgrade started until the new install is created
```

Comment:
what do you mean by "created"? because it's not created at this point, no? you mean available?